src/hotspot/share/runtime/os.cpp

```diff
@@ -1174,10 +1174,18 @@ bool os::is_readable_range(const void* from, const void* to) {
 // moved from debug.cpp (used to be find()) but still called from there
 // The verbose parameter is only set by the debug code in one case
 void os::print_location(outputStream* st, intptr_t x, bool verbose) {
+
+  // Trying to determine what a register refers to is inherently risky, so
+  // in case it fails we first print out the raw value. This leads to some
+  // duplication with the various `print` functions used below, but will aid
+  // with debugging things if we fail to decode the location.
+  st->print(INTPTR_FORMAT ": ", x);
+  st->flush();
```

Comment:
Nit: flush is not necessary, and strictly speaking should not be done by this function, since it depends on under which conditions the printing is done. In error handling, the caller must make sure that printing is unbuffered. And it is: we use unbuffered io (fdStream) which needs no flushing and where flush() is a noop.

So, I'd just remove the explicit flush() here.

Reply:
Okay I will remove that flush(). I just wanted to try and guarantee the raw value would get printed.